### DISCUSSION TITLE
正式從地球表面消失！微軟IE瀏覽器2023年2月全面停用

### DISCUSSION THREAD

**Discussion by 32844583** [1101-GS3073-C/Python-Community-GS3073-C#507] [General]
### 明年微軟IE瀏覽器的停用以及其影響
大約半年前，微軟宣布將停止在 Windows 10 系統上支援發布 27 年的 IE（Internet Explorer）瀏覽器，鼓勵用戶改用自己的 Edge 瀏覽器。然而，現在宣布 IE 將在 2023 年 2 月 14 日徹底消失，因為它將透過新的 Edge 瀏覽器的更新而刪除，而不是先前提到的 Windows 系統更新。在 2022 年 6 月的報告中，據指出日本有近半數的公司仍使用 IE 瀏覽器，因此如果 IE 徹底消失，日本將會是受到衝擊最大的國家。

[https://tw.stock.yahoo.com/news/%E6%AD%A3%E5%BC%8F%E5%BE%9E%E5%9C%B0%E7%90%83%E8%A1%A8%E9%9D%A2%E6%B6%88%E5%A4%B1-%E5%BE%AE%E8%BB%9Fie%E7%80%8F%E8%A6%BD%E5%99%A82023%E5%B9%B42%E6%9C%88%E5%85%A8%E9%9D%A2%E5%81%9C%E7%94%A8-112015504.html](url)

**Comment by manipulateed** [comment]
幸好我早早就開始用edge 我覺得這個瀏覽器超好用的啦～推薦給大家，這確實是一個非常方便好看簡潔的瀏覽器

**Comment by NcuJerryKao** [comment]
我已經忘記他了耶 不過現在想來好懷念喔 小時候都是用IE瀏覽器
好在看到你的貼文 近期來用一下懷舊 明年二月就沒了

**Reply by NcuJerryKao** [reply]
![ie擷取](https://user-images.githubusercontent.com/113414249/209363553-830264b6-b11d-4dab-a89f-82cd72111c68.PNG)
想懷舊的我 看到直接翻桌 現在電腦打ie 就能看到 並可直接開啟 打算從現在開始 設定IE為 預設瀏覽器

**Comment by williamyen0623** [comment]
時代眼淚，我們懷念他（IE），但我是開IE起來會翻桌誒，好慢哈哈

**Comment by Ben6uo** [comment]
只能逆舊的不去新的不來 現在有了edge都不錯了

**Comment by limitx0** [comment]
我會想念IE的
現在開始偶爾用IE來瀏覽網頁好了

**Comment by Yilin20021016** [comment]
原來過了那麼9了 我的當年都是他給的

**Comment by Raywei0214** [comment]
可能我從一開始接觸電腦就用chorm的關係 感覺用edge有點不太適應

**Comment by awkward-willy** [comment]
IE 因為老舊其實會導致很多近代的網路技術難以運行，如今正式走入歷史或許能掀起一陣網頁技術的改革，讓相對陳舊的網頁得到更新，或許是一個相當不錯的福音。

**Comment by larry40305** [comment]
我們不得不面對一個非常尷尬的事實，那就是，那麼，深入的探討IE，是釐清一切的關鍵。
